cmd/infra/aws/create_iam.go

```diff
@@ -45,6 +46,7 @@ type CreateIAMOptions struct {
 
 	additionalIAMTags      []*iam.Tag
 	CreateKarpenterRoleARN bool
+	UseROSAManagedPolicies bool
```

Comment:
Please add flags to supply these vars when invoking the `create infra aws` command outside of `create cluster aws`